README.md

```diff
@@ -1 +1,40 @@
-# dynavity
\ No newline at end of file
+# Dynavity
+
+
+## Setting up the project
+
+This project uses [XcodeGen](https://github.com/yonaskolb/XcodeGen) to generate Xcode project files. As such, a prerequisite is to install XcodeGen.
+
+1. Install XcodeGen
+```sh
+brew install XcodeGen
+```
+
+2. Open `Dynavity/settings.yml.sample` and fill in your actual DEVELOPMENT_TEAM ID.
+One way to find this is by going to the Build settings and choosing your team, then viewing the source file of `project.pbxproj` to get your team ID.
+
+3. Rename `Dynavity/settings.yml.sample` to `Dynavity/settings.yml`
+```sh
+mv Dynavity/settings.yml.sample Dynavity/settings.yml
```

Comment:
Might be better to use `cp` instead since `Dynavity/settings.yml.sample` is being version controlled.